dfds_ds_toolbox/analysis/plotting.py

```diff
@@ -204,30 +199,28 @@ def plot_roc_curve(data_train: pd.DataFrame, data_test: pd.DataFrame, label: str
     fpr_train, tpr_train, _ = roc_curve(data_train.target, data_train.predProba)
     roc_auc_train = roc_auc_score(data_train.target, data_train.predProba)
 
-    f, ax1 = plt.subplots(1)
+    fig, ax = plt.subplots(1)
```

Comment:
Is the (1) necessary? We can remove it since in the other functions we are not specifying it